scripts/pg-drop

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/env python3
```

Comment:
this is also needed for 201911. as this is python3 does it mean you will have to create a dedicated PR for 201911 which is not python 3?

Reply:
yes, I will create dedicated PR for 201911